packages/terra-site/package.json

```diff
@@ -24,7 +24,28 @@
   "dependencies": {
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
-    "terra-markdown": "^0.0.0"
+    "react-intl": "^2.2.3",
+    "react-router": "^3.0.2",
+    "terra-application": "^0.2.0",
```

Comment:
Update all terra dependencies on 0.x versions to use `0.x` as version in package.json